lib/scarlet/helpers/nick_helper.rb

```diff
@@ -0,0 +1,57 @@
+class Scarlet
+  module NickHelper
+    def handle_special_nick(name)
+      case name
+      when ':me'
+        sender.nick
+      when ':you'
+        server.current_nick
+      else
+        name
+      end
+    end
+
+    def find_nicks(names)
+      names.words.map do |name|
+        name = handle_special_nick(name)
+        Scarlet::Nick.first(nick: name)
+      end
+    end
+
+    def find_nick(name)
+      find_nicks(name).first
```

Comment:
If we're assuming ActiveRecord parity, `find` raises an error if more than one match was found (instead of silently hiding the error).